spec/requests/form_controller_spec.rb

```diff
@@ -135,6 +135,32 @@
       end
     end
 
+    context "when owning organisation doesn't have any managing agents" do
+      let(:params) do
+        {
+          id: lettings_log.id,
+          lettings_log: {
+            page: "stock_owner",
+            owning_organisation_id: managing_organisation.id,
+          },
+        }
+      end
+
+      before do
+        lettings_log.update!(owning_organisation: nil, created_by: nil, managing_organisation: nil)
+        lettings_log.reload
+      end
+
+      it "sets managing organisation to owning organisation" do
+        post "/lettings-logs/#{lettings_log.id}/stock-owner", params: params
```

Comment:
Can a path helper be used for this?

Reply:
Similar answer to this: https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/1716#discussion_r1238704714